README.md

```diff
@@ -77,6 +77,25 @@ If you want to create a package with all the dependencies bundled, you can run `
 
 Every patch is expected to adhere to our coding style, which is checked automatically by Travis. You can install the checkers locally using `npm run install-dependencies`, and then run the checks using `npm run check` before submitting a pull request. There is also `npm run fix`, that will attempt to fix the formatting.
 
+## Dockerizing selfoss
+
+Dockerfile for Selfoss RSS aggregator is bundled in the repository.
+
+Selfoss config is mounted in a separate volume, so your custom settings should survive reboot.
```

Comment:
Is that property of Docker or docker-compose?

Reply:
> Is that property of Docker or docker-compose?

Yes, we can mount volumes via docker

Reply:
But does not that need some extra CLI flags passed to `docker`?

Reply:
> But does not that need some extra CLI flags passed to `docker`?

If we use docker-compose then I think no

Reply:
Yeah, then I think it should be made clear that it is docker-compose that creates the volumes. Or show how to do it using plain `docker` invocation.

Reply:
> Yeah, then I think it should be made clear that it is docker-compose that creates the volumes. Or show how to do it using plain `docker` invocation.

Sure, I'll do it, and also, I'll add the volume support in the dockerfile itself. then it won't require anything additional via docker run also.

Reply:
> I can't figure out that why isn't it taking my config.ini into account then. may be some mouting/linking issue?
> The Dockerfile specifies the VOLUME for the config directory, so the directory will already be created inside. Then in the entrypoint we check if config.ini is present and create it if not.
> 
> So this can be used with or without docker-compose.
> 
> Running docker run directly even without the -v option will still create the volume, only it will not be named, but docker will still remember which volume belongs to which container and will use it next time you run the same container. Using docker-compose only allows you to have more flexibility with the volumes, such as you can easily create another container that uses the same volume and runs a backup on it, etc. Or you can easily join this container and its volumes with another containers in a swarm (which is why I was even looking into selfoss in the first place - I wanted to add an rss aggregator container into my existing project with many other containers)

Reply:
@akash07k Can you join the gitter room? I can help you debug the config issue, chatroom is a better tool for that. I'll be online for the next hour or two

Reply:
> I'm sorry, I just came online and checked your comment. are you still online? and I haven't joined any gitter chatroom untill now, is there any link to join?
> Please excuse me for this but I have only heard about gitter but never used it. Though, I'm keen to try it. :-)

> @akash07k Can you join the gitter room? I can help you debug the config issue, chatroom is a better tool for that. I'll be online for the next hour or two

Reply:
yes I'm here now: https://gitter.im/fossar/selfoss you will access it with your github account, just confirm the permissions

Reply:
Alternately, you can use [Matrix/Riot](https://matrix.to/#/#selfoss:matrix.org), or [IRC](https://irc.gitter.im/) to connect to the room, if you prefer.